programs/ssl/ssl_mail_client.c

```diff
@@ -356,9 +356,15 @@ int main( int argc, char *argv[] )
     int ret = 1, len;
     int exit_code = MBEDTLS_EXIT_FAILURE;
     mbedtls_net_context server_fd;
-    unsigned char buf[1024];
 #if defined(MBEDTLS_BASE64_C)
     unsigned char base[1024];
+    /* buf is used as destination buffer in snprintf calls with format:
+     * "%s\r\n". '%s' is substituted by string in base. Hence, size of
+     * buf should atleast be size of base plus 2 bytes for \r & \n characters.
```

Comment:
"should be at least the size of base plus 2 bytes for the \r and \n characters"

We replace "&" with "and" to avoid confusion this with bitwise AND.